dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/Topics.java

```diff
@@ -40,4 +40,8 @@
     String TRACKER_TOPIC_NAME = "dhis2.topic.tracker";
 
     String TRACKER_IMPORT_JOB_TOPIC_NAME = "dhis2.jobs.tracker";
+
+    String TRACKER_IMPORT_NOTIFICATION_TOPIC_NAME = "dhis2.jobs.tracker.notifications";
```

Comment:
This are really queues, it would be better to create a QUEUE constant class for queues